packages/api/models/Operator.ts

```diff
@@ -20,19 +13,18 @@ export enum AppEnvAccess {
 export interface Operator {
   display_name: string;
   email: string;
-  access: AccessLevel;
+  access: string;
 }
 
-export const operatorFields: ExhaustiveAttributeMap<Operator> = {
-  display_name: null,
-  email: null,
-  access: null,
-};
-
 export interface PortfolioOperator extends Operator {
-  access: AccessLevel.PORTFOLIO_ADMINISTRATOR;
+  access: PortfolioAccess.PORTFOLIO_ADMINISTRATOR;
 }
 
 export interface AppEnvOperator extends Operator {
-  access: AccessLevel.ADMINISTRATOR | AccessLevel.CONTRIBUTOR | AccessLevel.READ_ONLY;
+  access: AppEnvAccess.ADMINISTRATOR | AppEnvAccess.CONTRIBUTOR | AppEnvAccess.READ_ONLY;
 }
+export const operatorFields: ExhaustiveAttributeMap<Operator> = {
```

Comment:
Since `Operator` no longer has all the required properties the type used by `ExhaustiveAttributeMap` will need to change to accommdate for both `PortfolioOperator` and `AppEnvoperator`. I would suggest something like
```typescript
type Operators = PortfolioOperator | AppEnvOperator;
export const operatorFields: ExhaustiveAttributeMap<Operators>
```

Reply:
This was incorporated.